<launch>
    <arg name="attempts_count" default="1"/>
    <arg name="output" default="results.csv"/>

    <node name="drone_fsm" pkg="quadrotors_landing_vision" type="drone_fsm.py" respawn="false" output="screen">
        <rosparam file="$(find quadrotors_landing_vision)/config/drone/drone_fsm.yaml" command="load"/>
        <param name="attempts_count" type="int" value="$(arg attempts_count)" />
        <param name="output" type="str" value="$(arg output)" />
    </node>

</launch>
